src/test/java/com/tictactoe/ConfigurationTest.java

```diff
@@ -0,0 +1,5 @@
+package com.tictactoe;
+
+public class ConfigurationTest {
+
+}
```

Comment:
If you don't have any tests in the file, you should probably just delete it for now. You can always recreate it later